misk/web/@misk/common/src/styles/blueprint-core-2.3.1.min.css

```diff
@@ -0,0 +1,6 @@
+@charset "UTF-8";/*!

```

Comment:
off topic: why is this checked in and not fetched from NPM?

Reply:
Found a way to just pull from NPM, fixing...